apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/tildeling/TildelSaksbehandler.tsx

```diff
@@ -35,9 +35,9 @@ export const TildelSaksbehandler = (props: {
     <Button icon={<PersonIcon />} variant="tertiary" size="small" onClick={tildelSaksbehandlerWrapper}>
       Tildel meg
     </Button>,
-    () => (
+    (err) => (
       <Alert variant="error" size="small">
-        Tildeling feilet
+        {err.code === 'OPPGAVEN_HAR_ALLEREDE_SAKSBEHANDLER' ? 'Allerede tildelt' : 'Tildeling feilet'}
```

Comment:
Noen som har en grei måte å legge på tooltip på en alert? Tenkte det kunne være en grei måte å få med den lange forklaringen og løsningsbeskrivelsen Mari så for seg :)

Reply:
Ser det er gjort noe på OMS:
import { OmstillingsstoenadToolTip } from '~components/behandling/beregne/OmstillingsstoenadToolTip'
Passer det med noe sånt her?